src/main/java/com/example/AllPermutationPrime.java

```diff
@@ -0,0 +1,52 @@
+package com.example;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Készíts egy olyan függvényt, aminek
+ * bemenő paramétere n,
+ * és válaszként kilistázza azokat a számokat 1-től n-ig,
+ * amelyeknek az összes körkörös permutációja prím.
+ */
+public class AllPermutationPrime {
+
+    public static boolean isPrime(int n) {
+        if (n < 2) {
+            return false;
+        }
+        boolean prime = true;
+        int divider = 2;
+        int limit = (int) Math.sqrt(n) + 1;
+        while ((divider < limit) && prime) {
```

Comment:
It's just really a nitpick, you don't really need the `prime` boolean, you could do something like this:

```
while(diviede < limit) {
 if(n % divider == 0) {
 return false;
 }
 divider++;
}
return true
```

To me it looks simpler but it's no a big deal.